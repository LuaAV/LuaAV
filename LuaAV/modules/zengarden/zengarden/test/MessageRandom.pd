#N canvas 814 394 450 300 10;
#X obj 94 226 random 5;
#X obj 94 262 print;
#X obj 136 154 until;
#X msg 129 130 10;
#X obj 84 24 loadbang;
#X msg 227 150 20;
#X obj 82 85 t b b b b;
#X obj 161 30 bng 15 250 50 0 empty empty empty 17 7 0 10 -262144 -1
-1;
#X msg 64 160 seed 100;
#X connect 0 0 1 0;
#X connect 2 0 0 0;
#X connect 3 0 2 0;
#X connect 4 0 6 0;
#X connect 5 0 0 1;
#X connect 6 0 3 0;
#X connect 6 1 5 0;
#X connect 6 2 3 0;
#X connect 6 3 8 0;
#X connect 7 0 6 0;
#X connect 8 0 0 0;
