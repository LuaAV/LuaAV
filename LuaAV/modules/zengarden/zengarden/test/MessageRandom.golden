[@ 0.000ms] print: 0
[@ 0.000ms] print: 0
[@ 0.000ms] print: 3
[@ 0.000ms] print: 0
[@ 0.000ms] print: 2
[@ 0.000ms] print: 4
[@ 0.000ms] print: 2
[@ 0.000ms] print: 2
[@ 0.000ms] print: 2
[@ 0.000ms] print: 2
[@ 0.000ms] print: 17
[@ 0.000ms] print: 16
[@ 0.000ms] print: 15
[@ 0.000ms] print: 4
[@ 0.000ms] print: 11
[@ 0.000ms] print: 16
[@ 0.000ms] print: 9
[@ 0.000ms] print: 2
[@ 0.000ms] print: 12
[@ 0.000ms] print: 4
